packages/libz.rb

```diff
@@ -0,0 +1,29 @@
+require 'package'
+
+class Libz < Package
```

Comment:
I didn't like the `zlibpkg` name either but `zlib` is probably a better name since that is the project name.  It's an easy change, right?

Reply:
Renaming packages is not a simple fix.  You also need to consider how to remove the older version, otherwise, the metadata hangs around and can cause a potential problem.

Reply:
This was gone over before. `zlib` is a reserved keyword in Ruby: https://ruby-doc.org/stdlib-2.6.1/libdoc/zlib/rdoc/Zlib.html

Reply:
You didn't seem to care with GCC and all the fake packages that you're removing.

Reply:
It'd be nice to update `tools/core_packages.txt` also since this is a core package.

Reply:
If I remember correctly, wasn't it YOU who suggested all these changes to gcc in the first place?

Reply:
Not the ones you made. You just copied the already bad package for gcc7 into gcc8 and changed the numbers.